cosmos_test.go

```diff
@@ -1302,23 +1299,20 @@ func TestHandleTimeout(t *testing.T) {
 
 func TestHandleTimeout_Abort(t *testing.T) {
 	// GIVEN
-	cosmos := cosmosImpl{
-		logger:       zerolog.New(os.Stdout).Level(zerolog.DebugLevel),
-		retryTimeout: time.Millisecond * 50,
-	}
-	done := make(chan bool)
+	retryTimeout := time.Millisecond * 50
+	done := make(chan struct{})
 
 	timedOut := false
 	closed := false
 	mu := sync.Mutex{}
 
 	// WHEN
 	go func() {
-		timedOutChan := cosmos.handleTimeout(done)
+		timedOutChan := handleTimeout(done, retryTimeout, zerolog.Nop())
 
-		for isTimedOut := range timedOutChan {
+		for range timedOutChan {
 			mu.Lock()
-			timedOut = isTimedOut
+			timedOut = true
```

Comment:
now timedOut is always true whether something was sent on the timedOutChan or not